Earthfile

```diff
@@ -334,7 +257,7 @@ base-image:
     ARG KAIROS_VERSION=$(cat ./GIT_VERSION)
 
     IF [ "$FRAMEWORK_VERSION" = "" ]
-        ARG _FRAMEWORK_VERSION=master
+        ARG _FRAMEWORK_VERSION=main
     ELSE IF [ "$FRAMEWORK_VERSION" = "git" ]
         ARG _FRAMEWORK_VERSION=$VERSION
```

Comment:
This is not going to work anymore, is it?

Reply:
Shoot, but of course not, great catch!

Reply:
@jimmykarily ok I made it so it uses the latest stable